core-java/src/main/java/org/spine3/util/Methods.java

```diff
@@ -63,8 +63,8 @@ public static String getFullMethodName(Method method) {
      * @throws ClassCastException if the first parameter isn't a class implementing {@link Message}
```

Comment:
Please remove the link to `Class` from the parameter description. Just `the class` would be enough.